### DISCUSSION TITLE
Install Package

### DISCUSSION THREAD

**Discussion by hammerjoe** [sqm-autorate/sqm-autorate#158] [General] [answered]
Hi,

I wads wondering if anyone has made a package that can be used to install it in the Luci Gui directly, or can tell me how it can be done?

**Comment by chas-iot** [comment, marked as answer]
This software will not run on OpenWRT 22.x as we have not managed to convince the core devs to upgrade a required package to avoid a crash.

Unfortunately, this software is not packaged in a way that allows it to be installed under Luci, and must be installed from the router command line. The detailed installation instructions are at https://github.com/sqm-autorate/sqm-autorate#installation
